cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -698,7 +698,7 @@ func postCreate(ctx context.Context, clusterAccess *access.Adapter) error {
 func saveConfig(talosConfigObj *clientconfig.Config) (err error) {
 	c, err := clientconfig.Open(talosconfig)
 	if err != nil {
-		return err
```

Comment:
@TimJones we missed it, but @DmitriyMV noticed it: `%s` should be `%w` to allow proper wrapping

Reply:
@smira @DmitriyMV Thanks, fixed in #5629